### PR TITLE
Prevent hover state on buffer icon  for custom skins

### PR DESCRIPTION
**Pull request by DanFerrer**
### This PR will...

Prevent hover state from being triggered when the user has a custom player skin and hovers on the buffer icon when the player is buffering.

### Why is this Pull Request needed?

Buffer icon should remain stateless when hovered on.

#### Addresses Issue(s):

JW8-659

### CODE REVIEW

File: src/js/view/utils/skin.js [hunk 1]
```diff
@@ -142,11 +142,14 @@ export function handleColorOverrides(playerId, skin) {
             addStyle([
                 // Toggle and menu button active colors
                 '.jw-button-color.jw-toggle:not(.jw-icon-cast)',
-                '.jw-button-color:hover:not(.jw-icon-cast)',
                 '.jw-button-color:focus:not(.jw-icon-cast)',
                 '.jw-button-color.jw-toggle.jw-off:hover:not(.jw-icon-cast)'
             ], 'color', config.iconsActive);
 
+            addStyle([
+                ':not(.jw-state-buffering) .jw-button-color:hover:not(.jw-icon-cast)'
```
Comment (karimJWP) [inline src/js/view/utils/skin.js]: @DanFerrer would this prevent all icons from going to active state when hovered over ? I think the buffer icon is the only one that should be prevented

Comment (DanFerrer) [reply src/js/view/utils/skin.js]: @karimJWP Possibly, but the buffering icon is the only one that's shown when the player state is buffering. All other svgs with the class jw-button-color are hidden.

Comment (egreaves) [reply src/js/view/utils/skin.js]: @karimJWP - @DanFerrer is right in that it wouldn't. `:not(.jw-state-buffering)` ensures no hover color is applied when the player is in a buffering state.

**Comment by egreaves** [comment]
@DanFerrer - this is also a problem for the default player, but it's less noticeable because of the transition from `rgba(255,255,255,0.8)` to `#fff`. Take a look [here](http://player-pr-test-jenkins.longtailvideo.com/builds/10374/archive/test/view/).

File: src/js/view/utils/skin.js (hunk 1 above)
Comment (egreaves) [reply src/js/view/utils/skin.js]: @karimJWP you're right, actually. Dan's addressing your feedback now.

**Pull request merged**